python/data_structures/queue.py

```diff
@@ -1,12 +1,45 @@
+
+from data_structures.linked_list import Node
+from data_structures.invalid_operation_error import InvalidOperationError
+
+
 class Queue:
     """
     Put docstring here
     """
 
-    def __init__(self):
-        # initialization here
-        pass
+    def __init__(self, front = None):
+        self.front = None
+        self.rear = None
```

Comment:
You're accepting front as an argument here and not using it. Did you mean to have `self.front = front` on line 12?